src/stale-prs.ts

```diff
@@ -340,6 +342,28 @@ export class StalePrFinder {
 
     return ret;
   }
+
+  /**
+   * Returns true if the PR has been reviewed or commented on by a core member.
+   */
+  private async memberReviewed(pull_number: number): Promise<Boolean> {
+    const reviews = (await this.client.paginate(this.client.rest.pulls.listReviews, { ...this.repo, pull_number }));
+
+    const comments = await this.client.paginate(this.client.rest.issues.listComments, {
+      ...this.repo,
+      issue_number: pull_number,
+    });
+
+    // Reviews by team members
+    // Filtering out instances where submitted_at is empty
+    const memberReviews = reviews.filter(r => r.author_association === 'MEMBER').filter(r => r.submitted_at);
+
+    // Comments by team members
+    // Filtering out instances where submitted_at is empty
+    const memberComments = comments.filter(r => r.author_association === 'MEMBER').filter(r => r.submitted_at);
+
+    return memberComments.length > 0 || memberReviews.length > 0;
```

Comment:
If we are looking for either reviews or comments, can we do them one at a time. Why do we need to request all of the reviews and all of the comments all at once? Say we find a comment from a core team member, then we do not need to make a request for the reviews.

Reply:
Good point, I'll change this to only check reviews if after confirming there are no core member comments.